string_search/string_search.java

```diff
@@ -0,0 +1,36 @@
+package string_search;
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.util.Scanner;
+
+public class string_search {
+
+    public static void main(String[] args) {
```

Comment:
Could be extended with case insensitivity

Reply:
It was an optional task